### PR TITLE
xUnit logging support

### PR DESCRIPTION
**Pull request by thePantz**
## Description

> Changes for #227 
- Added project `Boa.Constrictor.Xunit` for items that improve [xUnit](https://xunit.net/) support
- Added project `Boa.Constrictor.Xunit.UnitTests`
- Added XunitLogger to support the use of `ITestOutputHelper`
- Added `XunitLoggerTests`

## Testing

> Built and executed unit tests. Based my test scenarios off of `ConsoleLoggerTests`

## Checklist

- [X] I agree to follow Boa Constrictor's [Code of Conduct](https://q2ebanking.github.io/boa-constrictor/contributing/code-of-conduct/).
- [X] I read Boa Constrictor's [Contributing Code](https://q2ebanking.github.io/boa-constrictor/contributing/contributing-code/) guide.
- [X] I successfully built the .NET solution with no errors or new warnings.
- [X] I successfully ran both the unit tests and the example tests.
- [X] I updated the [appropriate changelogs](CHANGELOG.md) with concise descriptions of these changes.
- [X] I added documentation for these changes (if appropriate).

### CODE REVIEW

File: Boa.Constrictor.Xunit.UnitTests/Logging/Loggers/XunitLoggerTest.cs [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+using FluentAssertions;
+using Moq;
+using Microsoft.VisualStudio.TestPlatform.Utilities;
+using Boa.Constrictor.Screenplay;
```
Comment (AutomationPanda) [inline Boa.Constrictor.Xunit.UnitTests/Logging/Loggers/XunitLoggerTest.cs:13]: Pet peeve, but please alphabetize all imports.

File: Boa.Constrictor.Xunit.UnitTests/Logging/Loggers/XunitLoggerTest.cs [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+        #endregion
+
+        #region Tests
+
+        [Fact]
+        public void Close()
+        {
+            Logger.Info("hello");
+            Logger.Info("moto");
```
Comment (AutomationPanda) [inline Boa.Constrictor.Xunit.UnitTests/Logging/Loggers/XunitLoggerTest.cs:78]: I love it.

File: Boa.Constrictor.Xunit.UnitTests/Boa.Constrictor.Xunit.UnitTests.csproj [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="FluentAssertions" Version="6.8.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
+    <PackageReference Include="Moq" Version="4.18.2" />
+    <PackageReference Include="xunit" Version="2.4.2" />
```
Comment (AutomationPanda) [inline Boa.Constrictor.Xunit.UnitTests/Boa.Constrictor.Xunit.UnitTests.csproj:15]: Note: Other unit test projects use NUnit, but it is proper for this one (for obvious reasons) to use xUnit.net.

**Review by AutomationPanda — CHANGES_REQUESTED** [review]
Excellent work. In addition to the few minor requests, please also add a `Boa.Constrictor.Xunit` package to the `nuget-push.yml` GitHub workflow so we can publish this package automatically once this pull request is merged. (Don't forget to update the changelog with the package version, too.

**Comment by thePantz** [comment]
@AutomationPanda Awesome! Thanks for the feedback! I see I made some copy pasta 😅I'm busy for the next couple days but I'll circle back to this on Friday

File: Boa.Constrictor.Xunit/Boa.Constrictor.Xunit.csproj [hunk 4]
```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+<PropertyGroup>
+    <TargetFrameworks>net5.0;net7.0;netstandard2.0</TargetFrameworks>
+    <Version>3.0.3</Version>
+    <Authors>Keith Tremorin, Pandy Knight and the PrecisionLender SETs</Authors>
```
Comment (AutomationPanda) [inline Boa.Constrictor.Xunit/Boa.Constrictor.Xunit.csproj:6]: Righteous.

**Review by AutomationPanda — APPROVED** [review]
Excellent work, @thePantz! Thank you so much for your contribution to Boa Constrictor. Let me know if you'd like me to mail you some official Boa Constrictor stickers!

**Comment by thePantz** [comment]
@AutomationPanda I'd love some stickers!

**Comment by CoffeeAtBedtime** [comment]
👏

**Pull request merged**